metadata-ingestion/src/datahub/ingestion/source/state/stale_entity_removal_handler.py

```diff
@@ -217,7 +213,6 @@ def create_checkpoint(self) -> Optional[Checkpoint]:
                 pipeline_name=self.pipeline_name,
                 platform_instance_id=self.source.get_platform_instance_id(),
                 run_id=self.run_id,
-                config=cast(ConfigModel, self.config),
```

Comment:
What risk do we run by removing this? Does this mean that previously changing a recipe wil create a new checkpoint state completely?

Reply:
Each run of the ingestion recipe will create a new checkpoint state. All this does is not include the raw config as part of that checkpoint